tests/integration/fixtures/split_and_sample_data/splitter_test_cases_and_fixtures.py

```diff
@@ -113,6 +144,47 @@ def get_unique_sorted_test_column_values(
 
         return column_values[:limit]
 
+    def get_unique_sorted_test_multi_column_values(
+        self,
+        reverse: Optional[bool] = False,
+        limit: Optional[int] = None,
+    ) -> List[dict]:
+        multi_column_values: List[dict] = self.get_test_multi_column_values()
+        multi_column_values = sorted(
+            multi_column_values,
+            key=lambda element: sum(
+                map(
+                    ord,
+                    hashlib.md5(
+                        str(tuple(zip(element.keys(), element.values()))).encode(
+                            "utf-8"
+                        )
+                    ).hexdigest(),
+                )
+            ),
+            reverse=reverse,
+        )
+
+        unique_multi_column_values: List[dict] = []
+
+        hash_codes: List[str] = []
+        hash_code: str
+        dictionary_element: dict
+        for dictionary_element in multi_column_values:
+            hash_code = hashlib.md5(
```

Comment:
This is really interesting. Is the md5 hashing so that the sorting is always unique?

Reply:
@Shinnnyshinshin Yes!  Nice way to add "hashability" property, isn't it?! :-). Note that in the `Domain` object, which is `DotDict` we add the "dunder" `hash()` method to accomplish the same (here we just do this for the dictionary "on a side").